data/json/professions.json

```
@@ -694,6 +694,50 @@
       "female": [ "panties", "sports_bra" ]
     }
   },
+  {
+    "type": "profession",
+    "id": "combat_engineer",
+    "name": "Combat Engineer",
+    "description": "A simple job. Keep the army moving. You built roads, you built bridges, you cleared minefields, you demolished buildings.  And, if need be, you built fortifications.  Your secondary job however, was being a rifleman.",
```

```suggestion
    "description": "A simple job.  Keep the army moving.  You built roads, you built bridges, you cleared minefields, you demolished buildings.  And, if need be, you built fortifications.  Your secondary job however, was being a rifleman.",
```

As you can see in the error message [here](https://travis-ci.org/github/CleverRaven/Cataclysm-DDA/jobs/742654089#L1237) (or the github actions annotation right below this line)